### ISSUE TITLE
[BUG] Inventory gets messed up when opening a Satellite Terminal.

### ISSUE THREAD

**Issue by Splutty** [zmaster587/AdvancedRocketry#1827]
## Version of Advanced Rocketry
> AdvancedRocketry-1.12.2-1.7.0-209-universal

## Have you verified this is an issue in the latest unstable build
- [x] N

## Version of LibVulpes
> LibVulpes-1.12.2-0.4.2-74-universal

## Version of Minecraft
> 1.12.2

## Version of Forge
> forge-1.12.2-14.23.5.2847-universal

## Does this occur without other mods installed
[X] No Clue
- Running Enigmatica 2 Expert, I can't remove mods from my server without breaking literally everything.

## Description of the problem
> See screenshots. If I open a satellite terminal part of my displayed inventory (but not my actual inventory) gets filled up with satellite chips. Ranges from 0 to 6+. Picking up anything in the inventory resets the display to what the inventory should actually look like.

https://i.imgur.com/05MqNta.png
https://i.imgur.com/P9wBD32.png

**Comment by Splutty** [comment]
Referenced: https://github.com/zmaster587/AdvancedRocketry/issues/1280

**Comment by Splutty** [comment]
I've tried finding a relation between terminal, inventory, etc. But so far I've failed.

I have 12 terminals, 4 for each data type, and the amount of "Duplicated" satellite chips showing is different for each terminal.

The extra satellite chips shown in the inventory are however duplicates of the one in the actual terminal I'm looking at.

The amount of extras shown isn't even consistent per terminal over time it appears.

On my 'oldest' 3 terminals it's 8,9,8 extras, then the next 3 is 5,6,2 (which was 5,6,4 when I checked 10 minutes ago), etc. There doesn't seem to be a logical progression or reason. This is all with the same inventory to start off with.
